src/Firebase/Exception/ApiException.php

```diff
@@ -58,7 +57,7 @@ public function getResponse()
 
     private static function getTargetClassFromStatusCode($code): string
     {
-        if (\in_array($code, [StatusCode::STATUS_UNAUTHORIZED, StatusCode::STATUS_FORBIDDEN], true)) {
+        if (\in_array($code, [401, 403], true)) {
```

Comment:
What is the reason to remove the usage of the constants?

Reply:
It's been quite some time, but I probably wanted to remove the transient dependency on the `fig/http-message-util` package (which is available through the inclusion of `kreait/firebase-tokens`). Removing that package would break this class.

I'm currently planning on a rework of both packages, so the issue might or might not apply in the future any more.